supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/api/ActionCommands.scala

```diff
@@ -146,14 +146,17 @@ object ActionCommands {
     s"grep -E -q '$lookupKeysAsString' /cromwell_root/stderr ; echo $$? > /cromwell_root/memory_retry_rc"
   }
 
-  def multiLineCommand(commandString: String) = {
+  def multiLineCommandTransformer(shell: String)(commandString: String): String = {
     val randomUuid = UUID.randomUUID().toString
     val withBashShebang = s"#!/bin/bash\n\n$commandString"
     val base64EncodedScript = Base64.encodeBase64String(withBashShebang.getBytes)
     val scriptPath = s"/tmp/$randomUuid.sh"
 
-      s"""python -c 'import base64; print(base64.b64decode("$base64EncodedScript"));' > $scriptPath && """ +
+    s"""python -c 'import base64; print(base64.b64decode("$base64EncodedScript"));' > $scriptPath && """ +
       s"chmod u+x $scriptPath && " +
-      s"sh $scriptPath"
+      s"$shell $scriptPath"
   }
+
+  def multiLineCommand(commandString: String): String = multiLineCommandTransformer("sh")(commandString)
+  def multiLineBinBashCommand(commandString: String): String = multiLineCommandTransformer("/bin/bash")(commandString)
```

Comment:
What new functionality requires bash?

Reply:
The `if [ ... ]` syntax. Normally I'd try to find another way to express it except we're guaranteed to be on the cloudSDK image which has bash already.

I also noticed that the entire `multiLineCommand` explicitly specifies `#!/bin/bash` at the top, but this seems to the immediately be ignored and the command is passed to `sh` 🤷‍♂️